internal/langserver/handlers/command/module_list.go

```diff
@@ -0,0 +1,145 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/creachadair/jrpc2/code"
+	lsctx "github.com/hashicorp/terraform-ls/internal/context"
+	"github.com/hashicorp/terraform-ls/internal/langserver/cmd"
+	"github.com/hashicorp/terraform-ls/internal/terraform/datadir"
+	"github.com/hashicorp/terraform-ls/internal/uri"
+)
+
+const moduleListVersion = 0
+
+type moduleListResponse struct {
+	FormatVersion int               `json:"v"`
+	Modules       []terraformModule `json:"modules"`
+}
+
+type terraformModule struct {
```

Comment:
```suggestion
const moduleCallsVersion = 0

type moduleCallsResponse struct {
	FormatVersion int               `json:"v"`
	ModuleCalls       []moduleCall `json:"module_calls"`
}

type moduleCall struct {
```

I think this better represents what the command returns as per https://www.terraform.io/docs/language/modules/syntax.html#calling-a-child-module